Lib/vanilla/test/testAll.py

```diff
@@ -15,8 +15,8 @@
 
 import objc
 objc.setVerbose(True)
-
-vanillaPath = os.path.dirname(os.path.dirname(os.path.dirname(vanilla.__file__)))
+vanillaPath = NSString.stringWithString_(vanilla.__file__).stringByResolvingSymlinksInPath()
```

Comment:
Wouldn't `vanillaPath = os.path.realpath(vanilla.__file__)` do the same thing?

Reply:
I tried to find how that would be done in python and google didn't tell me. 
Your code works fine. I’ll change it.